recipes/toolchain3/conda-forge-cc

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
```

Comment:
This LICENSE was for the whole spack codebase. I can ask the spack devs whether they'd like to re-license just this file if there's a need.